src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -57,6 +63,12 @@
       <NgenArchitecture>All</NgenArchitecture>
       <NgenPriority>3</NgenPriority>
     </ProjectReference>
+    <ProjectReference Include="..\Razor\RazorVisualStudio.csproj">
+      <Project>{0c0eeb55-4b6d-4f2b-b0bb-b9eb2ba9e980}</Project>
+      <Name>RazorVisualStudio</Name>
+      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
+      <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
```

Comment:
Do you want to add it for RazorServiceHub too?  I'm not sure it's worth it, since I assume these binaries are tiny and not loaded during VS startup/project load.